eradicate.py

```diff
@@ -28,176 +28,199 @@
 import io
 import os
 import re
-import sys
 import tokenize
 
 __version__ = '1.0'
 
+class Eradicator:
```

Comment:
We use `python2`, so don't forget the `(object)` part

Reply:
Oh yeah .. the relic py2 😋